### ISSUE TITLE
Unable to run "Convert To..." build

### ISSUE THREAD

**Issue by morozgrafix** [SublimeText/AAAPackageDev#51]
Fresh install of AAAPackageDev through Package Control:Install Package on OS X 10.10.1, Sublime 2.0.2 and Python 2.7.9.

Attempting to convert simple test.YAML-tmLanguage syntax definition to test.tmLanguage and getting following error in the when inspecting console:
```
Traceback (most recent call last):
  File "/Users/sergeym/Library/Application Support/Sublime Text 2/Packages/AAAPackageDev/Lib/sublime_lib/__init__.py", line 62, in run_
    self.typ.run_(self, *args)
  File "./sublime_plugin.py", line 339, in run_
  File "./file_conversion.py", line 128, in run
  File "/Users/sergeym/Library/Application Support/Sublime Text 2/Packages/AAAPackageDev/Lib/sublime_lib/view/output_panel.py", line 120, in __init__
    self.set_path(path, file_regex, line_regex)
  File "/Users/sergeym/Library/Application Support/Sublime Text 2/Packages/AAAPackageDev/Lib/sublime_lib/view/output_panel.py", line 131, in set_path
    self.set_regex(file_regex, line_regex)
  File "/Users/sergeym/Library/Application Support/Sublime Text 2/Packages/AAAPackageDev/Lib/sublime_lib/view/output_panel.py", line 155, in set_regex
    self.view.sel().add_all(sel)
Boost.Python.ArgumentError: Python argument types in
    RegionSet.add_all(RegionSet, list)
did not match C++ signature:
    add_all(SelectionSet {lvalue}, SelectionSet)
```
 
Have tried to search for the solution but couldn't find anything. Any help on the matter is appreciated.

**Comment by FichteFoll** [comment]
I can solve this by tomorrow. It's an issue with some code I commited a few days ago and didn't consider not to be working with ST2 (since that has higher type restrictions on API calls as the sublime_plugin module is not provided as Python code).

**Comment by morozgrafix** [comment]
Thanks for taking a look at it and confirming that this is a known issue.

**Comment by calculuswhiz** [comment]
Has this been fixed? Because I'm still getting the exact same error. Is there something I should do?

**Comment by morozgrafix** [comment]
I can confirm that it has been fixed in 1.0.3 (latest at the time of writing this comment). Assuming you have installed it via Package Control. Go to Package Control: List Packages and see what version you have installed, if you aren't on 1.0.3 you can go through Package Control: Package Upgrade and see if you can bring your version of AAAPackageDev to 1.0.3 (maybe worth restarting Sublime Text after upgrade). Hope this helps.

**Comment by calculuswhiz** [comment]
Perfect, just had to restart sublime. Thanks!
